src/link/output.c

```diff
@@ -175,9 +175,16 @@ static void writeBank(struct SortedSection *bankSections, uint16_t baseOffset,
 		bankSections = bankSections->next;
 	}
 
-	while (offset < size) {
-		putc(overlayFile ? getc(overlayFile) : padValue, outputFile);
-		offset++;
+	if (!disablePadding) {
```

Comment:
I wouldn't particularly write an `if` that has both a negated condition and an `else`, but maybe that's just me.

Reply:
Removed the `else` as per other comment.